schema/objects/transactions/change_event/StakeholderStatusChangeEvent.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/objects/transactions/change_event/StakeholderStatusChangeEvent.schema.json",
+  "title": "Object - Stakeholder Status Change Event",
+  "description": "Object describing a change event for the activity status of this stakeholder",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/primitives/objects/Object.schema.json"
+    },
+    {
+      "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/primitives/objects/transactions/Transaction.schema.json"
+    },
+    {
+      "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/primitives/objects/transactions/change_event/StakeholderChangeEvent.schema.json"
+    }
+  ],
+  "properties": {
+    "object_type": {
+      "const": "CE_STAKEHOLDER_STATUS"
+    },
+    "new_status": {
+      "description": "Denoting the beginning of this activity status on the change date",
+      "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/enums/StakeholderStatusType.schema.json"
+    },
```

Comment:
since there can only be one status for a given stakeholder, this doesn't use the "started" / "ended" approach that we took with relationships, but rather just "new" -- what is the new status after this event?